scripts/typos_rollout_check.py

```
@@ -0,0 +1,284 @@
+    if local_overlay.exists():
+        phrases.update(_phrases(_document(local_overlay)))
+    return PhrasePolicy(
+        phrase_corrections=tuple(sorted(phrases.items())),
+        ignore_patterns=_strings(_table(generated, "default"), "extend-ignore-re"),
+        excluded_files=_strings(_table(generated, "files"), "extend-exclude"),
+    )
+
+
+def _tracked(repository: Path) -> tuple[Path, ...]:
+    """Return tracked paths in deterministic order."""
+    raw = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)